src/OrbitQt/MoveFilesDialog.ui

```diff
@@ -24,76 +21,59 @@
   </property>
   <property name="minimumSize">
    <size>
-    <width>471</width>
-    <height>260</height>
+    <width>300</width>
+    <height>200</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
-    <width>471</width>
-    <height>260</height>
+    <width>1280</width>
+    <height>960</height>
    </size>
   </property>
   <property name="windowTitle">
-   <string>Moving files</string>
+   <string>Moving files to Documents</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
-  <widget class="QPushButton" name="closeButton">
-   <property name="enabled">
-    <bool>false</bool>
-   </property>
-   <property name="geometry">
-    <rect>
-     <x>380</x>
-     <y>230</y>
-     <width>80</width>
-     <height>23</height>
-    </rect>
-   </property>
-   <property name="text">
-    <string>Close</string>
-   </property>
-  </widget>
-  <widget class="QLabel" name="label">
-   <property name="enabled">
-    <bool>true</bool>
-   </property>
-   <property name="geometry">
-    <rect>
-     <x>10</x>
-     <y>10</y>
-     <width>168</width>
-     <height>16</height>
-    </rect>
-   </property>
-   <property name="text">
-    <string>Moving files to new location</string>
-   </property>
-  </widget>
-  <widget class="QTextEdit" name="log">
-   <property name="enabled">
-    <bool>true</bool>
-   </property>
-   <property name="geometry">
-    <rect>
-     <x>10</x>
-     <y>30</y>
-     <width>451</width>
-     <height>192</height>
-    </rect>
-   </property>
-   <property name="sizePolicy">
-    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
-     <horstretch>0</horstretch>
-     <verstretch>0</verstretch>
-    </sizepolicy>
-   </property>
-   <property name="readOnly">
-    <bool>true</bool>
-   </property>
-  </widget>
+  <layout class="QVBoxLayout">
```

Comment:
Thanks! I spent non-trivial amount of time trying to apply a layout :)

Reply:
It's indeed not obvious how to do it in the designer (right click the the container > "Lay out" > "Lay Out Vertically", and NOT dragging the desired layout from the widgets on the left).